core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -147,6 +148,230 @@ class ConsumerBounceTest extends IntegrationTestHarness with Logging {
     }
   }
 
+  /**
+   * Consumer is closed while cluster is healthy. Consumer should complete pending offset commits
+   * and leave group. New consumer instance should be able join group and start consuming from
+   * last committed offset.
+   */
+  @Test
+  def testClose() {
```

Comment:
Have you investigated why `close()` is taking longer in the bounce test? Would be helpful to know which case we're hitting that is causing the delay.